test/connection_test.rb

```diff
@@ -411,6 +411,14 @@ def test_init_with_block_yields_connection
     assert_equal '/omnom', conn.path_prefix
   end
 
+  def test_respond_to
+    assert Faraday.default_connection.respond_to?(:get)
```

Comment:
> "why would you add assertions"

The concept of the test was, "this object delegates method calls to the `default_connection` object" rather than "this object responds to some arbitrary method".

If you prefer the latter, I'm happy to change it.